discovery-provider/src/api/v1/playlists.py

```diff
@@ -102,7 +102,9 @@ def get_playlist(
     args = format_get_playlists_args(current_user_id, playlist_id, route, with_users)
     playlists = get_playlists(args)
     if playlists:
-        return extend_playlist(playlists[0])
+        extendedPlaylist = extend_playlist(playlists[0])
```

Comment:
sup with this?

Reply:
I want to return `playlist_contents` (with the same data as `added_timestamps`) from the playlist endpoint, so that users can add/remove/reorder without having to know about `added_timestamps`. But doing so in `extend_playlist` causes an error downstream somewhere in ES logic, presumably because playlist_contents already exists but in a different format. There are some other instances of use modifying an entity before returning the response in the route, so went with this approach